src/mpid/ch4/netmod/ofi/ofi_rma.h

```diff
@@ -671,20 +671,109 @@ MPL_STATIC_INLINE_PREFIX int MPIDI_OFI_do_accumulate(const void *origin_addr,
                                                      MPI_Datatype target_datatype,
                                                      MPI_Op op, MPIR_Win * win,
                                                      MPIDI_av_entry_t * addr,
+                                                     MPIDI_winattr_t winattr,
                                                      MPIR_Request ** sigreq)
 {
     int mpi_errno = MPI_SUCCESS;
-    size_t origin_bytes;
+    int target_contig, origin_contig;
+    size_t target_bytes, origin_bytes, target_extent;
```

Comment:
We prefer `MPI_Aint` for these internal variables for consistency.

Reply:
It has to be changed for all RMA operations in this file in order to use the same common subroutines such as `MPIDI_OFI_query_acc_atomic_support`, `MPIDI_OFI_prepare_target_mr`, `MPIDI_OFI_check_acc_order_size`. Thus, I made the change in a separate commit.